src/routes/relay/relay.service.ts

```diff
@@ -11,6 +16,7 @@ export class RelayService {
   constructor(
     @Inject(IConfigurationService) configurationService: IConfigurationService,
     private readonly relayRepository: RelayRepository,
+    private readonly httpErrorFactory: HttpErrorFactory,
```

Comment:
We don't need this injected.

Reply:
Removed in 7bd3e3d48d8d989ace3f9a8163817e9662ebb736.